cmd/exchange_template/test_file.tmpl

```diff
@@ -36,6 +36,7 @@ func TestMain(m *testing.M) {
 	{{ if .WS }} exchCfg.API.AuthenticatedWebsocketSupport = true {{ end }}
 	exchCfg.API.Credentials.Key = apiKey
 	exchCfg.API.Credentials.Secret = apiSecret
+	exchCfg.Enabled = true

```

Comment:
Is there anything else that should be changed? I only ask because there is no other `TestMain` in `/exchanges/` that has this present and they work with authenticated endpoints. If this is an issue for them, could you update all of those?

Reply:
Never mind! Vague error messages led me here, but the fundamental issue was no assets/pairs being set in configtest.json on initial generation, which is the more graceful fix in general.

Maybe we could leave a comment about that somewhere in this tool, or in the files generated by it?

Reply:
All G! Yeah that's fine. Adding some details to suggest that you should populate configtest.json with asset/pairs before running tests would be good. Whether it's in ADD_NEW_EXCHANGE.md, commentary in the tool or otherwise I'm happy with. We're lacking in documentation, so any extra details is appreciated

Reply:
I've now done so in the printed output from running the tool. Please let me know what you think of the wording there.